redirects.yaml

```diff
@@ -1,3 +1,3 @@
-docs: http://docs.ubuntu.com/maas/2.2/en/
```

Comment:
Actually, could we make this point to `https://docs.ubuntu.com/maas/`, and then update here - https://github.com/canonical-websites/docs.ubuntu.com/blob/master/redirects.yaml#L9 - to point that to the correct version?

Updating to the latest version shouldn't really require changes on maas.io itself.